helm/cas-bciers/templates/reporting/route.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
 {{ include "cas-bciers.labels" . | indent 4 }}
   annotations:
     haproxy.router.openshift.io/balance: roundrobin
-    haproxy.router.openshift.io/rewrite-target: /
+    haproxy.router.openshift.io/rewrite-target: {{ .Values.registrationFrontend.route.path }}
```

Comment:
should that be `reportingFrontend` ?

Reply:
Yup, a mistype when rewriting from the testing branch. Fixed with e3c33d7.